doc/api/crypto.md

```diff
@@ -172,14 +172,14 @@ console.log(encrypted);
 // Prints: ca981be48e90867604588e75d04feabb63cc007a8f8ad89b10616ed84d815504
 ```
 
-### cipher.final([output_encoding])
+### cipher.final([outputEncoding])
```

Comment:
These changes break "hard links" and as such I do not think we should do this unless necessary.